### PR TITLE
typo-fixes

### PR DESCRIPTION
**Pull request by Uziel302**
#### Metadata

typo-fixes

### CODE REVIEW

File: files/en-us/web/css/@property/syntax/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ syntax: '<color>'; /* accepts a color */
 
 syntax: '<length> | <percentage>'; /* accepts lengths or percentages but not calc expressions with a combination of the two */
 
-syntax: 'small | medium | large'; /* accepts one of these values set as custom idents. */
+syntax: 'small | medium | large'; /* accepts one of these values set as custom indents. */
```
Comment (Josh-Cena) [inline files/en-us/web/css/@property/syntax/index.md:25]: "idents" is the right word. It's short for "identifier". See https://developer.mozilla.org/en-US/docs/Web/CSS/custom-ident, for example

Comment (Uziel302) [reply files/en-us/web/css/@property/syntax/index.md]: removed this change

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thank you!

**Pull request merged**